docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,178 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
 last_update:
-  date: 30 June 2023
+  date: 4 Jan 2024
 ---
 
-Rollups play a crucial part in providing next-generation scaling on Tezos. This page gives a technical introduction to Smart Rollups, their optimistic nature, and an intro to developing your own WASM kernel.
+import LucidDiagram from '@site/src/components/LucidDiagram';
 
-## Examples
+Smart Rollups play a crucial part in providing high scalability on Tezos.
+They handle transactions and logic in a separate environment that doesn't need to follow all of the same rules as the main Tezos network.
+For this reason, they are not subject to the same block delays and data space constraints as the main Tezos network.
```

Comment:
This is also a bit misleading. 
Rollups by themselves are still subject to the same block delays of the L1 network. In order to be processed, data that allows to retrieve L2 transactions (either directly or by using a Data availability solution, see below) must be posted to the rollup inbox on the L1. 

Data space constraints require a data availability solution which is separate from rollups. Smart rollups, however, integrate with two data availability solutions: the Data Availability Layer (DAL) and Data Availability Committees (DACs), the latter via a mechanism called the reveal data channel.

Reply:
How about "They handle logic in a separate environment that can run transactions at a much higher rate and can use larger amounts of data than the main Tezos network."